glass/grf/_transformations.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from ._core import ArrayT, corr, dcorr, icorr
+
+
+@dataclass
+class Normal:
+    """Transformation for normal fields."""
+
+    def __call__(self, x: ArrayT, _var: float, /) -> ArrayT:
+        """Return *x* unchanged."""
+        return x
+
+
+@dataclass
+class Lognormal:
+    """Transformation for lognormal fields."""
+
+    lamda: float = 1.0
+
+    def __call__(self, x: ArrayT, var: float, /) -> ArrayT:
+        """Transform *x* into a lognormal field."""
+        xp = x.__array_namespace__()
+        x = xp.expm1(x - var / 2)
+        if self.lamda != 1.0:
+            x = self.lamda * x
+        return x
+
+
+@dataclass
+class SquaredNormal:
+    """
+    Transformation for squared normal fields.
+
+    References
+    ----------
+    .. [1] https://arxiv.org/abs/2408.16903
```

Comment:
```suggestion
    * [1] https://arxiv.org/abs/2408.16903
```

Reply:
Thanks. This should probably get moved to the bibliography in #497.